clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -84,13 +84,13 @@ class ClioWebClient private[client] (
 
     /*
      * Reusable `Source` which will run the given `HttpRequest` over the connection
-     * flow to the Clio server, erroring out early if the request fails to complete
+     * flow to the Clio server, erroring out early if the response fails to arrive
      * within the request timeout.
      */
     val responseSource = Source
       .single(requestWithCreds)
       .via(connectionFlow)
-      .completionTimeout(requestTimeout)
+      .initialTimeout(requestTimeout)
```

Comment:
Would it make sense to set an idle timeout as well?

Reply:
👍 I'll try it out. I originally tried setting one here but hit the same issue with the HTTP connection going idle after 30 seconds, but it should(?) work if I stick it after the retry logic.